openmls/src/framing/mod.rs

```diff
@@ -39,15 +39,68 @@ pub use validation::*;
 mod test_framing;
 
 /// Wire format of MLS messages.
+///
+/// # MLS Presentation Language
```

Comment:
I think we still call it TLS presentation language in the spec because we never defined what the MLS presentation language is

Reply:
So maybe just "Presentation Language" as in "2.1. Presentation Language"?

Reply:
I've removed the section everywhere.